### PR TITLE
[DO_NOT_MERGE][3.x]Liveness probe

### PR DESCRIPTION
**Pull request by rhcarvalho**
To discuss (at least):
* what will be the endpoints URLs
* status codes

For now I'm intentionally not handling exceptions to let Django's 500 page come up with debugging info.

@bparees @mfojtik PTAL

### CODE REVIEW

**Comment by bparees** [comment]
lgtm with a squash.

**Comment by rhcarvalho** [comment]
@bparees the readiness probe is not working

**Comment by rhcarvalho** [comment]
Not adding readiness probe for now. Wait for  https://github.com/GoogleCloudPlatform/kubernetes/issues/9798.

**Comment by rhcarvalho** [comment]
This goes post-3.0.

File: openshift/templates/django.json [hunk 1]
```diff
@@ -186,7 +187,13 @@
                     "name": "DJANGO_SECRET_KEY",
                     "value": "${DJANGO_SECRET_KEY}"
                   }
-                ]
+                ],
+                "livenessProbe": {
+                  "httpGet": {
+                    "path": "/_health/live",
+                    "port": "web"
```
Comment (ewolinetz) [inline openshift/templates/django.json]: working with @ironcladlou it seems that currently this doesn't resolve correctly for the liveness probe and returns a port of 0.  work around is to specify the value of 'web'

**Comment by rhcarvalho** [comment]
Closing this since we had probes added in #32.

**Pull request closed without merging**